mytoyota/client.py

```diff
@@ -540,3 +544,58 @@ async def get_trip_json(self, vin: str, trip_id: str) -> str:
         """
         trip = await self.get_trip(vin, trip_id)
         return json.dumps(trip.raw_json, indent=3)
+
+    async def request_lock_vehicle(self, vin: str) -> VehicleLockUnlockActionResponse:
+        """Sends a lock command to the vehicle.
+
+        Args:
+            vin (str): Vehicle identification number.
+
+        Raises:
+            ToyotaLoginError: An error returned when updating token or invalid login information.
+            ToyotaInternalError: An error occurred when making a request.
+            ToyotaApiError: Toyota's API returned an error.
+        """
+        _LOGGER.debug(f"Locking {censor_vin(vin)}...")
+        raw_response = await self.api.set_lock_unlock_vehicle_endpoint(vin, "lock")
+        _LOGGER.debug(f"Locking {censor_vin(vin)}... {raw_response}")
+        response = VehicleLockUnlockActionResponse(raw_response)
+        return response
+
+    async def request_unlock_vehicle(self, vin: str) -> VehicleLockUnlockActionResponse:
+        """Send an unlock command to the vehicle.
+
+        Args:
+            vin (str): Vehicle identification number.
+
+        Raises:
+            ToyotaLoginError: An error returned when updating token or invalid login information.
+            ToyotaInternalError: An error occurred when making a request.
+            ToyotaApiError: Toyota's API returned an error.
+        """
+        _LOGGER.debug(f"Locking {censor_vin(vin)}...")
+        raw_response = await self.api.set_lock_unlock_vehicle_endpoint(vin, "unlock")
+        _LOGGER.debug(f"Locking {censor_vin(vin)}... {raw_response}")
+        response = VehicleLockUnlockActionResponse(raw_response)
+        return response
+
+    async def get_lock_request_status(

```

Comment:
´get_lock_status´ is simpler

Reply:
personally I prefer to be more explicit (the method gets the status of the request, rather than the status of the locks), but I've updated it fwiw. I suppose the functional signature/docstring is enough